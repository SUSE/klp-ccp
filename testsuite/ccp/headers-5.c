static void
#include "headers-5.h"
i(void)
{}

void pu_f(void)
{
	h();
}
