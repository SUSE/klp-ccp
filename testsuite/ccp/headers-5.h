g(void)
{}

static void h(void)
{}

static void
