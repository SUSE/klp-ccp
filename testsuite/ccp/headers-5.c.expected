/* klp-ccp: from testsuite/ccp/headers-5.c */
static void
#include "headers-5.h"
i(void)
{}

void klpp_pu_f(void)
{
	h();
}
